src/lib/mina_base/party.ml

```diff
@@ -522,6 +544,7 @@ module Body = struct
       ; Events.to_input events
       ; Events.to_input sequence_events
       ; Random_oracle_input.field call_data
+      ; Random_oracle_input.field use_full_commitment
```

Comment:
This needs to be `bitstring` to match the above, otherwise hashes will be inconsistent inside and outside of the circuit.